hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
@christinem I'll catch-up! Please, bare with me :+1: 

@Ian-Stewart-Binks I have the book "Clean Code" now, so please let me know if my code hearkens back to any of its chapters.

@david-yz-liu Sorry Professor, I thought Pull-requests were just for reviewing rough drafts, so I didn't put any final touches i.e. cleaning up code, documentation, comments, etc. I will do that now.